File: docs/snippets/read_score_app_secret.rb
require 'jwt'

authorization = JWT.encode({ request: params[:request] }, ENV['APP_SECRET'], 'HS512')
response      = Faraday.get('https://rg.traity.com/score',
                            nil,
                            'Authorization': "Bearer #{authorization}").body
score         = JSON.parse(response)

File: docs/snippets/read_score_app_secret.js
const axios = require('axios')
const jwt = require('jsonwebtoken')

const credentials = jwt.sign({ request: req.query.request }, process.env.APP_SECRET, {
  algorithm: 'HS512'
})
const scoreApiRes = await axios.get('https://rg.traity.com/score', {
  headers: { authorization: `Bearer ${credentials}` }
})
const score = scoreApiRes.data
